docker-process.el

```diff
@@ -91,7 +96,8 @@
       (aio-resolve promise
                    (lambda ()
                      (message nil)
-                     (message "Finished: %s" (process-name process))
+                     (when docker-message-commands
```

Comment:
Out of curiosity, what does `(message nil)` above/below do here?

Reply:
It hides the previous message otherwise the minibuffer spans multiple lines